internal/exec/stages/files/units.go

```diff
@@ -73,6 +75,7 @@ func (s stage) createUnits(config types.Config) error {
 // applies to the unit's dropins.
 func (s stage) writeSystemdUnit(unit types.Unit, runtime bool) error {
 	return s.Logger.LogOp(func() error {
+		relabeledDropinDir := false
```

Comment:
I'd just say `if len(unit.Dropins) != 0 { s.relabel; }` instead of keeping track of if you've done with this var.